lib/Transforms/SerializeSPIRV.cpp

```diff
@@ -0,0 +1,66 @@
+//===- SerializeSPIRV.cpp - SPIR-V serialize pass --------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// serialize MLIR SPIR-V module to SPIR-V binary
```

Comment:
\file

Reply:
this view is outdated,  we can see the change in the "file changed" like below
//===- SerializeSPIRV.cpp - SPIR-V serialize pass --------------*- C++ -*-===//
//
// Copyright 2022 Intel Corporation
// Part of the IMEX Project, under the Apache License v2.0 with LLVM Exceptions.
// See https://llvm.org/LICENSE.txt for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
//===----------------------------------------------------------------------===//
///
/// \file
/// This pass iterates all the SPIR-V modules in the top module and serializes
/// each SPIR-V module to SPIR-V binary and then attachs the binary blob as a
/// string attribute to the corresponding gpu module.
///
//===---------------

Reply:
Ok, my bad. Thanks!